pkg/config/config_template.yaml

```diff
@@ -1019,6 +1019,25 @@ api_key:
   #
   # integrations_logs_files_max_size
 
+
+  ## @param streaming - custom object - optional
+  ## This section allows you to configure streaming logs via remote config.
+  # streaming:
+
+    ## @param enable_streamlogs - bool - optional - default: true
+    ## @env DD_LOGS_CONFIG_STREAMING_ENABLE_STREAMLOGS - bool - optional - default: true  
+    ## Enable streaming logs via remote config.
+    # enable_streamlogs: true
```

Comment:
Does this mean we are always copying all logs to `streamlogs_log_file`?

Reply:
Yes, we don't have a filter feature yet so we are currently copying all logs from the stream-logs command to streamlogs_log_file.

Reply:
Would it be worth it to start with the feature off by default and run a few tests before enabling by default? \

I'm not what is the DD way of delivering new features to customers, but I can see that we would like to test something like this before enabling by default 😄

Reply:
I think i made some confusions here by defaulting it to true before. The correct default runtime setting value should be false and it should be adjusted according to how remote config set the value base on their config here:
https://github.com/DataDog/datadog-agent/pull/27026/files#diff-0728f6d5ed5d0de385153398bda70f3297aeabe049f9312c5bfcc709c4359ddeR304-R309

In this case, default is false, and if agent detects a signal from remote config that it wants to stream-logs then the agent will set the runtime to true and stream logs accordingly.

The main reason for why runtime setting is necessary is because remote config uses runtime setting as a mean to trigger actions without restarting the agent. 
Eg: they use log level run time to decide whether an info/debug flare is sent.

Additionally, I think it makes more sense that the agent would [consolidate streamlogs related configurable behaviors](https://github.com/DataDog/datadog-agent/pull/27026/files#diff-a77843a3a2d99bdc21000210a1c6a0fc99871774b33d15b6779389e8299aa794R1032-R1040) such as durations and pathing and future features such as log rotations, filter . If the agent were to receive those configurable options instead I think it would leave more room for network error.